pyproject.toml

```diff
@@ -41,6 +41,7 @@ ulid = "^1.1"
 
 # TODO: Remove this arbitrary python constraint once `sqlalchemy-bigquery` has done so.
 sqlalchemy-bigquery = { version = "1.9.0", python = "<3.13" }
+airbyte-api = {git = "https://github.com/airbytehq/airbyte-api-python-sdk.git", rev = "aj/manual_rename_dir"}
```

Comment:
TODO (`airbyte-api`):

- If possible, re-release the `airbyte-api` library so we can pin the dependency to a released version.
- Otherwise, pin to a specific commit so the reference does not break post-release of PyAirbyte.